apps/api/src/server/applications/subscriptions/subscriptions.routes.js

```diff
@@ -0,0 +1,127 @@
+import { Router as createRouter } from 'express';
+import { asyncHandler } from '../../middleware/async-handler.js';
+import {
+	getSubscription,
+	putSubscription,
+	updateSubscription
+} from './subscriptions.controller.js';
+import {
+	validateCreateSubscription,
+	validateGetSubscription,
+	validateUpdateSubscription
+} from './subscriptions.validators.js';
+
+const router = createRouter();
+
+router.get(
```

Comment:
Is the use only case for the get by case/email so that we can obtain the id for the delete (update) operation?  Fine if it is and probably re-usable in future.